spec/features/users/user_can_search_rdv_on_rdv_mairie_spec.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+describe "User can search rdv on rdv mairie" do
+  let(:now) { Time.zone.parse("2021-12-13 8:00") }
+
+  def json_response
+    JSON.parse(page.html)
+  end
+
+  before do
+    default_url_options[:host] = "http://www.rdv-mairie-test.localhost"
+  end
+
+  around do |example|
+    previous_auth_token = ENV["ANTS_API_AUTH_TOKEN"]
+    ENV["ANTS_API_AUTH_TOKEN"] = ""
+
+    example.run
+
+    ENV["ANTS_API_AUTH_TOKEN"] = previous_auth_token
+  end
+
+  before do
+    travel_to(now)
+    create(:plage_ouverture, :no_recurrence, first_day: now, motifs: [motif], lieu: lieu, organisation: organisation, start_time: Tod::TimeOfDay(9), end_time: Tod::TimeOfDay.new(10))
+  end
+
+  describe "default" do
```

Comment:
vu que c'est le seul groupe describe et qu'on n'a pas grand chose à dire pour le décrire, on peut juste enlever cette ligne et s'épargner un niveau d'indentation

Reply:
Done in 9bdfe74bee86a5e7a4dddb0f0d49b53cadd28fbf